master/pkg/tasks/task_command.go

```diff
@@ -81,6 +81,6 @@ func (s GenericCommandSpec) ToTaskSpec(
 	}
 
 	res.UseFluentLogging = true
-
+	res.TaskType = model.TaskTypeCommand
```

Comment:
generic command spec is actually for commands, shells, tensorboards and notebooks, since they used to always be the same

Reply:
Fixed